modules/chapter1/pages/section4.adoc

```diff
@@ -0,0 +1,408 @@
+= KServe - Model Serving
+
+RHOAI has two main components for serving 
+
+* KServe Serving for single-model serving
+* KServe ModelMesh for multi-model serving
+
+The two components are similar in that they retrieve models from S3 buckets and expose these models via an inferencing endpoint.
+However there are significant differences in how they work and the use cases they serve. 
+
+== Model Mesh
+In the multi-model approach serving pods can run one or more models simulateneously from the same pod. 
+As requests for a particular model are received the ModelMesh routing layer assigns the model to an existing serving pod. 
+In the serving pod a modelmesh-runtime-adapter retrieves the specified model and hands it over to the model adapter to serve.
+Once the model is available the modelmesh-runtime-adapter instructs the routing later to forward any requests related to the model to the serving endpoint.
+The modelmesh routing layer handles the dynamic nature of inferencing requests in that it can spin up additional model instances when load increase as well 
+as reducing the number of serving instances when load decreases. 
+
+The ModelMesh approach is useful for when customers are operating at a large scale with significant numbers of frequently changing models of different types. 
+
+We have covered *ModelMesh* previously so we'll just concentrate on KServe Serving here.
+
+== KServe Serving
+
+KServe Serving takes a different approach to ModelMesh. Using this approach a single model (hence the name) is served from a pod 
+and those pods are managed and controlled via OpenShift Serverless (KNative) and ingress traffic is handled by OpenShift Service Mesh. 
+Using KNative enables serving pods to be spun up and down, even to zero, depending on request load. Similar to ModelMesh the models are loaded from S3 buckets.
+
+
+image::Kserve-Serving.png[KServe Architecture]
+ref: https://kserve.github.io/website/master/images/kserve_layer.png
+
+
+At the time of writing the Red Hat OpenShift AI (2.8.0) has the following KServe Serving Runtimes:
+
+* OpenVINO Model Server which can serve the following model formats:
+    ** OpenVINO IR
+    ** ONNX
+    ** Tensorflow
+* Caikit TGIS
+* TGIS Standalone
```

Comment:
This is the first time we mention Caikit and TGIS if I am not wrong. It would be useful to add a one or two-sentence definition for each of them. Just a brief intro. You can tell students that they will find more details in section 5.